public/app/features/explore/Logs/LogsTable.tsx

```diff
@@ -74,93 +79,200 @@ export const LogsTable: React.FunctionComponent<Props> = (props) => {
         field.config = {
           ...field.config,
           custom: {
-            filterable: true,
             inspect: true,
+            filterable: true, // This sets the columns to be filterable
             ...field.config.custom,
+            // @todo it would be much more efficient to mutate the dataframe fields, instead of using transforms to create a filtered copy in rect state, this would require that the table component handles the filtering.
```

Comment:
```suggestion
            // @todo it would be much more efficient to mutate the dataframe fields, instead of using transforms to create a filtered copy in react state, this would require that the table component handles the filtering.
```